src/app/app-routing.module.ts

```diff
@@ -8,13 +8,17 @@ const routes: Routes = [
   },
   {
     path: 'auth',
-    loadChildren: () => import('./pages/auth/auth.module').then( m => m.AuthPageModule)
-  }
+    loadChildren: () => import('./pages/auth/auth.module').then(m => m.AuthPageModule)
+  },
+  {
+    path: 'sessions',
```

Comment:
use this router path name in a constant file and use

Reply:
done